python/ray/data/_internal/execution/streaming_executor_state.py

```diff
@@ -332,17 +336,22 @@ def process_completed_tasks(topology: Topology) -> None:
         while op.has_next():
             op_state.add_output(op.get_next())
 
+
+def postprocess_completed_tasks(topology: Topology) -> None:
```

Comment:
Let's rename this to `update_operator_states`.

`process_completed_tasks`'s comment says "Process any newly completed tasks and update operator state.". Now we move the "update state" part to this function.